website/views/donations.py

```diff
@@ -119,8 +119,16 @@ def webapp_view_donation():
         log(f"Exception when getting a campaign for a donations: {e}", severity="ERROR")
         return render_template("errors/403.html"), 403
 
+    logs_url = None
+    try:
+        trace_id = request.headers.get('X-Cloud-Trace-Context').split('/')[0]
+        logs_url = f'https://console.cloud.google.com/logs/query;query=trace%3D~%22projects%2F.*%2Ftraces%2F{trace_id}%22' if trace_id else None
```

Comment:
Confirmed that it points to default logging and default GCP project. 
Could we add the following in this query?
* `project`: GCP project id as defined in their `PROD_PROJ` / `STAGE_PROJECT` / `OPS_PROJECT`
* `resource.type` 
* `resource.labels.service_name`
* `resource.labels.location`

Reply:
I'm not opposed to `resource.type`, since we can hard-code that.

`project` and `resource.labels.location` would require [external HTTP queries](https://cloud.google.com/run/docs/container-contract#metadata-server) (in this case, to the _Metadata server_). _(To me, this feels like overkill.)_

Ditto for `service_name`, according to [this SO post](https://stackoverflow.com/questions/65628822/google-cloud-run-can-a-service-know-its-own-url).

Reply:
Exactly my concern for default project that is would be different from where they have emblem. That's the reason for asking to specify project and having the "View Logs" button direct the user to the correct `website` container should only be helpful not overkill. If more discussion is necessary, let's move this conversation offline.

Reply:
Not sure what's meant by an "external" query, the metadata server is a local service only available inside the sandbox of each Cloud Run container instance.

I think it would be good to have the GCP project available as a variable in the frontend for this and other debug tooling along these lines.

Service Name is available via an environment variable.

In terms of the query that I'd prefer to have in linking to logs, I'd rather not overly filter it: we know that if we filter to a trace ID, we will get all logs across all services and regions that are related to how that request was processed. That comprehensive view is part of the goal.

I will write a separate comment to address the complexity around trace headers.